lavamoat/browserify/beta/policy.json

```diff
@@ -1421,7 +1421,7 @@
       "packages": {
         "@metamask/scure-bip39": true,
         "@metamask/snaps-sdk>@metamask/key-tree>@metamask/utils": true,
-        "@ethereumjs/tx>ethereum-cryptography>@noble/curves": true,
+        "viem>@scure/bip32>@noble/curves": true,
```

Comment:
I'm not sure why this is changd :thinking: 
Viem is introduced in this PR as a test dependency (only used in our e2e)

https://www.npmjs.com/package/viem

cc @MetaMask/supply-chain does it make sense to you?

Reply:
thank you @leotm :pray: